plugins/task/checkin/src/Extension/Checkin.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * @package     Joomla.Plugins
+ * @subpackage  Task.Checkin
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Plugin\Task\Checkin\Extension;
+
+// Restrict direct access
+defined('_JEXEC') or die;
+
+use DateInterval;
+use Joomla\CMS\Application\CMSApplicationInterface;
+use Joomla\CMS\Plugin\CMSPlugin;
```

Comment:
```suggestion
use Joomla\CMS\Factory;
use Joomla\CMS\Plugin\CMSPlugin;
```
Was mentioned in my previous review comment that this should be added, too.